ziti/cmd/database/add_debug_admin.go

```diff
@@ -0,0 +1,129 @@
+/*
+	Copyright NetFoundry Inc.
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+	https://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+
+package database
+
+import (
+	"github.com/openziti/storage/boltz"
+	"github.com/openziti/ziti/common/eid"
+	"github.com/openziti/ziti/controller/change"
+	"github.com/openziti/ziti/controller/command"
+	"github.com/openziti/ziti/controller/db"
+	"github.com/openziti/ziti/controller/model"
+	"github.com/openziti/ziti/controller/network"
+	"github.com/openziti/ziti/controller/persistence"
+	"github.com/spf13/cobra"
+)
+
+func NewAddDebugAdminAction() *cobra.Command {
+	action := &addDebugAdminAction{}
+	return &cobra.Command{
+		Use:   "add-debug-admin </path/to/ziti-controller.db.file>",
+		Short: "Adds an admin (username: admin, password: admin) to the given database file for debugging purposes",
+		Args:  cobra.ExactArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+			action.run(args[0])
+		},
+	}
+}
+
+type addDebugAdminAction struct {
+	db       boltz.Db
+	stores   *db.Stores
+	managers *network.Managers
+}
+
+func (action *addDebugAdminAction) GetDb() boltz.Db {
+	return action.db
+}
+
+func (action *addDebugAdminAction) GetStores() *db.Stores {
+	return action.stores
+}
+
+func (action *addDebugAdminAction) GetManagers() *network.Managers {
+	return action.managers
+}
+
+func (action *addDebugAdminAction) noError(err error) {
+	if err != nil {
+		panic(err)
+	}
+}
+
+func (action *addDebugAdminAction) run(dbFile string) {
+	boltDb, err := db.Open(dbFile)
+	action.noError(err)
+
+	fabricStores, err := db.InitStores(boltDb)
+	action.noError(err)
+
+	dispatcher := &command.LocalDispatcher{
+		EncodeDecodeCommands: false,
+	}
+	controllers := network.NewManagers(nil, dispatcher, boltDb, fabricStores, nil)
+
+	dbProvider := &addDebugAdminAction{
+		db:       boltDb,
+		stores:   fabricStores,
+		managers: controllers,
+	}
+
+	stores, err := persistence.NewBoltStores(dbProvider)
+	action.noError(err)
+
+	id := "7dbd3fc9-e4c8-489a-ab8f-4bbb3d768f57"
+	ctx := change.New().SetChangeAuthorType("cli.debug-db").NewMutateContext()
+	err = dbProvider.GetDb().Update(ctx, func(ctx boltz.MutateContext) error {
+		tx := ctx.Tx()
+		identity, _ := stores.Identity.LoadOneById(tx, id)
+		if identity == nil {
```

Comment:
Does it make sense for this to update the existing one if present?

Reply:
I changed it to delete the existing identity if a debug-admin already exists.